lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -82,9 +82,8 @@ def unkeyword!(keyword_ids:)
       # Synchorize the taggings of this item with the ones in its mirrors. So all of them
       # are associated to the same tags
       def sync_mirrors_with_taggings
-        if is_mirrored_with_taggings
-          mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
-        end
+        return unless is_mirrored_with_taggings
+        mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
```

Comment:
;)

So turns out `is_mirrored_with_taggings` is an attr_accessor, not just a method. I could reconfigure it a bit to give us a predicate method too but I'd start to feel a bit like [this](http://i1.kym-cdn.com/photos/images/facebook/000/234/739/fa5.jpg) when trying to rename things like `is_mirrored_with_taggings`. Can we lay hands on anyone who has a fully fledged understanding of what this code is doing?